lib/container.js

```diff
@@ -364,7 +364,7 @@ export default class Container {
     });
   }
 
-  registerDevice(token, type) {
+  registerDevice(token, type, bundleName) {
```

Comment:
To incrementally polish our class doc. It is better to add a short description on what is the bundleName parameter is for.